src/base/generate-images-readme.md

```diff
@@ -0,0 +1,139 @@
+<!-- View this file with embedded videos on GitHub: -->
+<!-- https://github.com/yunity/karrot-frontend/blob/master/src/base/generate-images-readme.md -->
+
+# Generate Landingpage Images
+
+**TL:DR**
+- put new images in designated `_raws` folder
+- run `yarn generate-landing-images`
+- if you want to control the ordering of some images in a folder, go to `Landing.vue` and update `sortingArr` for that folder
+
+<br />
+
+## Requirements
+
+There is a `_raws` folder under `src/base/pages/images` with four subfolders where screenshots must be put.
+
+Requirements of each folder inside `_raws`:
+
+- `app-screenshots-browser`
+  - must contain exactly ***one*** image
+  - file-type: `png`
+  - file-name: `karrot-screenshot-browser.png`
+  - image dimensions: 2644 x 1726
+
+- `app-screenshots-phone`
+  - must contain exactly ***one*** image
+  - file-type: `png`
+  - file-name: `karrot-screenshot-phone.png`
+  - image dimensions: 750 x 1624
+
+- `feature-screenshots`
+  - must contain exactly ***three*** images
+  - file-type: `png`
+  - file-names:
+    - `karrot-feature-activities.png`
+    - `karrot-feature-groups.png`
+    - `karrot-feature-offers.png`
+
+- `random-imgs`
+  - no specific requirements
+  - images will be cropped to square -> pre-crop the image to the desired view
+  - if an image is smaller than 600x600, it will get scaled-up to that size
+
+<br />
+
+## Taking screenshots with browser devtools
+
+Screenshots for all folders (expect `random-imgs`) have to be taken with browser devtools. Here is an example of how it has to be done in Chrome. Firefox is about the same.
+
+<video controls="true" allowfullscreen="true" width="400">
+  <source src="assets/generate-images-readme/taking-screenshots-example.mp4" type="video/mp4">
+</video>
+
+https://user-images.githubusercontent.com/23213583/119812392-67513f00-bee8-11eb-9036-23b3dbab3d63.mp4
+
+<i>Example: taking screenshot for "karrot-screenshot-browser.png"</i>
+
+<br />
+
+<strong style="color: red;">NOTE:</strong> For each screenshot, make sure to
+- first go into device mode
+- select "Responsive" profile (not iPhone 6 etc)
+- set DPR to 2
+
+<br />
+
+**`app-screenshots-browser`** (see video above)
+  - set width/height to 1322 x 863
+  - take screenshot
+  - width/height of generated screenshot should be 2644 x 1726
+
+**`app-screenshots-phone`**
+  - set width/height to 375 x 812
+  - take screenshot
+  - width/height of generated screenshot should be 750 x 1624
+
+**`feature-screenshots`**
+- `karrot-feature-activities.png`
+  - set width/height to 1600 x 1600
+  - follow these steps:
+
+    <video controls="true" allowfullscreen="true" width="400">
+      <source src="assets/generate-images-readme/karrot-feature-activities.mp4" type="video/mp4">
+    </video>
+
+    https://user-images.githubusercontent.com/23213583/119814776-14c55200-beeb-11eb-9f19-fb5c81ed50e3.mp4
```

Comment:
I'm a bit confused about these two URLs here...  I'm assuming the `<video>` tag is the thing that is rendering the video [1], but there is also this other URL, which looks more like the URL when you upload a resource in a comment, etc.

I think I'd prefer it if the mp4s were kept _out_ of the repo, as it increases the repo size a lot, in a way we can never undo again. Currently size of `src` directory is 3.3mb, and this PR increases it to 40mb, of which 22mb is these mp4s. Can we just point the `src` to a GitHub uploaded URL? (I think the karrot logo in production still points to an image I uploaded in a random GitHub comment draft!).  

[1] https://github.com/yunity/karrot-frontend/blob/change/improve-landingpage-design/src/base/generate-images-readme.md

Reply:
mp4 files removed.

I thought, local files would be the way to go (works with the `video` tag). But then noticed the videos aren't showing up if viewing this file on github.com. Then uploaded the videos directly in the github edited md (the https://user-images links). I then kept both, so people viewing this md locally or on github can see a video-player.
But you can click on the video-link if viewing locally...

The rest of the size-increase is the `pages/images` folder: from 0.5mb to 15.5mb